.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Context
+
+<!-- Why do we need this PR? What was the reason that led you to make this change? -->
```

Comment:
I think it's a lot of extra noise that we don't need to read but the creator should be aware of. Just like comments in code, these comments further clarify what each section represents